docs/get-started/build-aspire-apps-with-python.md

```diff
@@ -160,33 +160,36 @@ Select the **Endpoints** link to open the `hello-python` endpoint in a new brows
 
 Stop the app host by pressing <kbd>Ctrl</kbd> + <kbd>C</kbd> in the terminal.
 
-## Add telemetry support.
+## Add telemetry support
 
-To add a bit of observability, add telemetry to help monitor the dependant Python app. In the Python project, add the following OpenTelemetry package as a dependency in the _requirements.txt_ file:
+To add a bit of observability, add telemetry to help monitor the dependant Python app. In the Python project, add the following **OpenTelemetry** packages as a dependency in the _requirements.txt_ file:
 
-:::code language="python" source="snippets/PythonSample/hello-python/requirements.txt" highlight="2-5":::
+:::code language="text" source="snippets/PythonSample/hello-python/requirements.txt" highlight="2-5":::
 
-The preceding requirement update, adds the OpenTelemetry package and the OTLP exporter. Next, re-install the Python app requirements into the virtual environment by running the following command:
+Next, reinstall the Python app requirements into the virtual environment by running the following command:
 
-```python
+```console
 python -m pip install -r requirements.txt
 ```
 
-The preceding command installs the OpenTelemetry package and the OTLP exporter, in the virtual environment. Update the Python app to include the OpenTelemetry code, by replacing the existing _main.py_ code with the following:
+The preceding command installs the **OpenTelemetry** package and the **OTLP** exporter, in the virtual environment. Update the Python app to include the **OpenTelemetry** code, by replacing the existing _main.py_ code with the following:
 
 :::code language="python" source="snippets/PythonSample/hello-python/main.py":::
 
-Update the app host project's _launchSettings.json_ file to include the `ASPIRE_ALLOW_UNSECURED_TRANSPORT` environment variable:
+Update the app host project's _launchSettings.json_ file to include the `ASPIRE_ALLOW_UNSECURED_TRANSPORT` environment variable under the `http` profile:
 
-:::code language="json" source="snippets/PythonSample/PythonSample.AppHost/Properties/launchSettings.json":::
+:::code language="json" source="snippets/PythonSample/PythonSample.AppHost/Properties/launchSettings.json" highlight="26":::
 
 The `ASPIRE_ALLOW_UNSECURED_TRANSPORT` variable is required because when running locally the OpenTelemetry client in Python rejects the local development certificate. Launch the _app host_ again:
 
 ```dotnetcli
-dotnet run --project ../PythonSample.AppHost/PythonSample.AppHost.csproj
+dotnet run --project ../PythonSample.AppHost/PythonSample.AppHost.csproj --launch-profile http
 ```
 
-Once the app host has launched navigate to the dashboard and note that in addition to console log output, structured logging is also being routed through to the dashboard.
+> [!IMPORTANT]
+> The .NET Aspire host must be run using HTTP instead of HTTPS. The **OpenTelemetry** library requires HTTP when running in a local dev environment.
```

Comment:
```suggestion
> The .NET Aspire app host must be run using HTTP instead of HTTPS. The **OpenTelemetry** library requires HTTP when running in a local dev environment.
```